### PR TITLE
Scala 2.13 support.

### PR DESCRIPTION
**Pull request by tzeman77**
Hi @zhongl,

please apply the following pull-request which adds support for Scala 2.13 and bumps couple of plugins/dependencies.

Thanks in advance,
/tomas

PS. many thanks for this great library!

### CODE REVIEW

**Comment by zhongl** [comment]
@tzeman77 Thanks for this PR!

Please fix ci build error by changing jdk option with **openjdk8**.

File: build.sbt [hunk 1]
```diff
@@ -1,13 +1,27 @@
 lazy val root = (project in file("."))
   .settings(basicSettings: _*)
+  .settings(
+    libraryDependencies ++= {
+      CrossVersion.partialVersion(scalaVersion.value) match {
+        case Some((2, n)) if n <= 12 =>
+	  List(compilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.full))
```
Comment (zhongl) [inline build.sbt:7]: Be careful with paradise `v2.1.1`, which is not for all scala `12.x`.  `2.1.0` maybe the better choice.

File: build.sbt [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
   .settings(dependencySettings: _*)
-  .settings(addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full))
 
 lazy val basicSettings = Seq(
   name := "config-annotation",
   organization := "com.wacai",
-  scalaVersion := "2.11.5",
-  crossScalaVersions := Seq(scalaVersion.value, "2.12.0"),
+  scalaVersion := "2.13.4",
+  crossScalaVersions := Seq(scalaVersion.value, "2.12.13", "2.11.12"),
```
Comment (zhongl) [inline build.sbt:24]: Please let scala versions in `.travis.yml` be the same with `crossScalaVersions`.

**Review by zhongl — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 1 above)
Comment (tzeman77) [reply build.sbt:7]: ok, I'll revert back to `2.1.0`; wil set scala 2.12 to `2.12.10` since this is the last version supported by the plugin.

**Review by zhongl — APPROVED** [review]

**Pull request merged**